Game/Resources/elevatorScene/elevatorScene_01.json

```diff
@@ -4524,6 +4524,82 @@
 			"scene" : false,
 			"static" : false,
 			"uid" : 1512566495
+		},
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/NewAssets/Textures/Rock/rock_albedo",
+				"metallic" : "Resources/NewAssets/Textures/Rock/rock_metallic",
+				"normal" : "Resources/NewAssets/Textures/Rock/rock_normal",
+				"roughness" : "Resources/NewAssets/Textures/Rock/rock_roughness"
+			},
+			"Mesh" : 
+			{
+				"model" : "Resources/VoltageScene/Models/VoltageCavein"
+			},
+			"children" : 
+			[
+				{
+					"Material" : 
+					{
+						"albedo" : "Resources/VoltageScene/Textures/Skull/Skull_Albedo",
+						"metallic" : "",
```

Comment:
Missing metallic and roughness textures.

Reply:
This means default texture